cluster/manifests/instana-poc/daemonset-instana.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: monitor
+    application: instana-agent
+    version: v1
+  name: instana-agent
+  namespace: kube-system
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      name: instana-agent
+      labels:
+        application: instana-agent
```

Comment:
This should have the version as well: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/external-dns/deployment.yaml#L19 (You'll have to use the `matchLabels:` stuff then as well)

Reply:
👍